releases/release-1.22/release-notes/maps/pr-102059-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 102059
+releasenote:
+  text: Fixed very rare volume corruption when a pod is deleted while kubelet is
```

Comment:
Enclose "pod" and "kubelet" in back-ticks ?

Reply:
I do not think that pod and kubelet should be back ticked in the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) is never done